workqueue.c

```diff
@@ -136,13 +136,17 @@ static void *worker_thread(void *data)
 
 	sk_out_assign(sw->sk_out);
 
+#ifndef __rtems__
 	if (wq->ops.nice) {
 		if (nice(wq->ops.nice) < 0) {
 			log_err("workqueue: nice %s\n", strerror(errno));
 			ret = 1;
 		}
 	}
-
+#else /* __rtems__ */
+	log_err("workqueue: nice %s\n", strerror(errno));
```

Comment:
`errno` isn't set here like for the original case. Maybe replace the message with something other. Like `workqueue: no nice in RTEMS\n`.